omero/sysadmins/unix/server-centos6.txt

```diff
@@ -0,0 +1,123 @@
+Installing Python 2.7 and Ice 3.5 on CentOS 6
+=============================================
+
+Running OMERO on CentOS 6 has a number of special requirements which
+deviate from the standard installation instructions.  The instructions
+below will set up Python 2.7 and Ice 3.5 on CentOS 6.
+
+Python 2.7
+----------
+
+CentOS 6 provides Python 2.6.  However, OMERO.web requires Python 2.7
```

Comment:
Or perhaps better put: there are 2 ways to read this. Perhaps clarify that "in order to use Django 1.8, Python 2.7 is required".